src/pld/gemini.c

```diff
@@ -14,44 +14,31 @@
 #include "pld.h"
 #include "helper/time_support.h"
 
-//#define LOCAL_BUILD
 //#define EMULATOR_BUILD
 
-#ifdef LOCAL_BUILD
-#define GEMINI_IDCODE         	0x20000913
-#define GEMINI_DEBUG_CONTROL  	0x80001028
-#define GEMINI_SPARE_REG      	0x800010f0
-#define GEMINI_SRAM_ADDRESS   	0x80002000
-#define GEMINI_LOAD_ADDRESS   	0x80002000
-#endif
-
 #ifdef EMULATOR_BUILD
-#define GEMINI_IDCODE         	0x1000563d
-#define GEMINI_DEBUG_CONTROL  	0xf1000028
-#define GEMINI_SPARE_REG      	0x800000f0
-#define GEMINI_SRAM_ADDRESS   	0x80001000
-#define GEMINI_LOAD_ADDRESS   	0x80001000
-#endif
-
-#if !defined(LOCAL_BUILD) && !defined(EMULATOR_BUILD)
-#define GEMINI_IDCODE         	0x1000563d
-#define GEMINI_DEBUG_CONTROL  	0xf1000028
-#define GEMINI_SPARE_REG      	0xf10000f0
-#define GEMINI_SRAM_ADDRESS   	0x80000000
-#define GEMINI_LOAD_ADDRESS   	0x00000000
+#define GEMINI_SPARE_REG		0x800000f0
+#define GEMINI_SRAM_ADDRESS		0x80001000
+#define GEMINI_LOAD_ADDRESS		0x80001000
+#else
+#define GEMINI_SPARE_REG		0xf10000f0
+#define GEMINI_SRAM_ADDRESS		0x80000000
+#define GEMINI_LOAD_ADDRESS		0x00000000
```

Comment:
they are not aligned

Reply:
I follow the OpenOCD C coding style as documented here: https://openocd.org/doc-release/doxygen/stylec.html

![image](https://user-images.githubusercontent.com/119840333/228748813-7b1214fa-f5da-4c1e-89ba-105c669d2c0b.png)